src/Aquifer.Jobs/SendNewContentEmails.cs

```diff
@@ -0,0 +1,100 @@
+    public async Task Run([TimerTrigger("%NewContentEmail:CronSchedule%")] TimerInfo timerInfo, CancellationToken ct)
+    {
+        var today = DateTime.Today;
+        var firstOfThisMonth = new DateTime(today.Year, today.Month, 1);
+        var firstOfLastMonth = firstOfThisMonth.AddMonths(-1);
+
+        var subscribers = await dbContext.ContentSubscribers.Where(cs => cs.Enabled)
+            .Select(cs => new SubscriberInfo
+            {
+                Name = cs.Name,
+                Email = cs.Email,
+                UnsubscribeId = cs.UnsubscribeId,
+                Languages = cs.ContentSubscriberLanguages.Select(csl => csl.Language),
+                ParentResources = cs.ContentSubscriberParentResources.Select(cspr => cspr.ParentResource)
+            })
+            .ToListAsync(ct);
+
+        var alLanguageIds = subscribers.SelectMany(x => x.Languages.Select(l => l.Id));
+        var allParentResourceIds = subscribers.SelectMany(x => x.ParentResources.Select(pr => pr.Id));
+
+        var allNewItems = await dbContext.ResourceContentVersions.Where(x =>
+                x.IsPublished &&
+                x.Updated >= firstOfLastMonth &&
+                x.Updated <= firstOfThisMonth &&
+                alLanguageIds.Contains(x.ResourceContent.LanguageId) &&
+                allParentResourceIds.Contains(x.ResourceContent.Resource.ParentResourceId))
+            .Select(x => new UpdatedParentResources{ParentResourceId = x.ResourceContent.Resource.ParentResourceId,
+                DisplayName = x.ResourceContent.Resource.ParentResource.DisplayName,
+                LanguageId = x.ResourceContent.LanguageId,
+                EnglishDisplay = x.ResourceContent.Language.EnglishDisplay})
+            .ToListAsync(ct);
+
+        var htmlTemplate = dbContext.EmailTemplates.Single(t => t.Id == (int)EmailTemplate.AquiferMarketingNotification);
+
+        foreach (var subscriber in subscribers)
+        {
+            var subscriberLanguageIds = subscriber.Languages.Select(l => l.Id);
+            var subscriberParentResourceIds = subscriber.ParentResources.Select(pr => pr.Id);
+
+            var newItems = allNewItems.Where(x =>
+                subscriberLanguageIds.Contains(x.LanguageId) && subscriberParentResourceIds.Contains(x.ParentResourceId)).ToList();
+
+            if (newItems.Count == 0)
+            {
+                continue;
+            }
+            var resourcesLanguages =
+                newItems.Aggregate("", (current, item) => current + $"{item.DisplayName} - {item.EnglishDisplay}<br />");
+            var htmlContent = htmlTemplate.Template
+                .Replace("[NAME]", subscriber.Name)
+                .Replace("[RESOURCES]", resourcesLanguages)
+                .Replace("[RESOURCE_LINK]", options.Value.MarketingEmail.ResourceLink)
+                .Replace("[UNSUBSCRIBE]",
+                    $"{options.Value.MarketingEmail.UnsubscribeBaseUrl}/marketing/unsubscribe/{subscriber.UnsubscribeId}?api-key=none");
```

Comment:
Seems like this should just be some sort of self root url, because it's not specific to marketing, it's the server's own self.